server/models/Collective.js

```diff
@@ -221,6 +221,7 @@ export default function(Sequelize, DataTypes) {
             const image = `https://logo.clearbit.com/${getDomain(this.website)}`;
             return image;
           }
+          return `https://ui-avatars.com/api/?rounded=true&name=${encodeURIComponent(this.name)}`;
```

Comment:
Shouldn't we only use ui-avatars for 'USER'?

Reply:
What's the impact of that change in the frontend?

Reply:
Good idea to only do it for `USER`.

It means that we will have to check for `ui-avatars.com` instead of `undefined` if we want to show a different default avatar on the frontend. But in the meantime, it will fix this: https://github.com/opencollective/opencollective/issues/1903